pllug-presentation-system/presentation-data/presentationelement.h

```diff
@@ -0,0 +1,14 @@
+#ifndef PRESENTATIONELEMENT_H
+#define PRESENTATIONELEMENT_H
+
+/*!
+ * \brief The PresentationElement class represents single element that could be placed
+ * on presentation slide.
+ */
+class PresentationElement
```

Comment:
Maybe, better name this class SlideElement, because this will be a build part of slides? What do you think?